lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -406,14 +407,24 @@ class MetroItinerary extends NarrativeItinerary {
             {!mini && (
               <ItineraryGrid className="itin-grid" role="group">
                 {/* TODO: a11y: add aria-label to parent element */}
-                <InvisibleHeader as={expanded && 'h2'}>
-                  <FormattedMessage
-                    id="components.MetroUI.itineraryDescription"
-                    values={{
-                      routes: getItineraryRoutes(itinerary, intl),
-                      time: formatDuration(itinerary.duration, intl)
-                    }}
-                  />
+                <InvisibleHeader as={expanded && 'h1'}>
+                  {String(transitRoutes) ? (
+                    <FormattedMessage
+                      id="components.MetroUI.itineraryDescription"
+                      values={{
+                        routes: transitRoutes,
+                        time: formatDuration(itinerary.duration, intl)
+                      }}
+                    />
+                  ) : (
+                    <FormattedMessage
+                      id="components.MetroUI.singleModeItineraryDescription"
+                      values={{
+                        mode: itinerary.legs[0].mode,
```

Comment:
Mode has to be translated. As written, it is the OTP mode enum value. For bike and walk, use for instance `common.modes.bike/walk` or `otpUi.ItineraryBody.travelByMode.bike/walk`.

Reply:
Good catch, sorry for missing that! Fixed in cbd2965